subprojects/docs/src/docs/release/notes.md

```diff
@@ -63,7 +63,18 @@ _Build time improvements using [Santa Tracker Android](https://github.com/gradle
 
 ## Configuration cache improvements
 
-TBD - load from cache performance improvements and reduced memory consumption for Android builds
+Gradle 6.6 introduced [configuration caching](userguide/configuration_cache.html) as an experimental feature.
+This release comes with performance and usability improvements for the configuration cache.
+
+Loading from the configuration cache is now faster and memory consumption during builds has been reduced, especially for Kotlin and Android builds.
+
+Starting with this Gradle version the configuration cache gets discarded when the build fails because problems have been found.
```

Comment:
Consider adding some context on why this is so important. For example, something like "Early adopters of configuration cache can use the command line output and HTML report for [troubleshooting](https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:troubleshooting). Previously, the configuration cache state was saved despite reported issues, which in some situation required manual cache invalidation. In this release, the configuration cache gets discarded when the build fails because of configuration cache problems. Note that you can still [ignore](https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:usage:ignore_problems) known issues."